unified-runtime/source/adapters/level_zero/v2/queue_immediate_in_order.cpp

```diff
@@ -908,16 +908,9 @@ ur_result_t ur_queue_immediate_in_order_t::enqueueGenericCommandListsExp(
 
   auto [pWaitEvents, numWaitEvents] =
       getWaitListView(commandListLocked, phEventWaitList, numEventsInWaitList);
-  // zeCommandListImmediateAppendCommandListsExp is not working with in-order
```

Comment:
I think for now we might need a check "if driver version is older than add sync", because CI won't have a new enough version yet.